scraper.py

```diff
@@ -0,0 +1,7 @@
+def excel_file_scraper(excel_file) -> list:
```

Comment:
1. get_file_content_could be better name
2. also `excel_file` sounds like a file object, but here you get string, so `filename` also would be better.

Reply:
And yeah, typing for arguments it's also a good one